tests/test_options/conftest.py

```diff
@@ -8,6 +9,8 @@
 @pytest.fixture()
 def option_parser():
     """Returns option parser that can be used for tests."""
+    if flake8_version_info > (5, ):
```

Comment:
I think it should be:

```python
if flake8_version_info > (5, ):
    parser = OptionManager(version=version.pkg_version, plugin_versions='', parents=[])
else:
    ...
```

Reply:
I tried that, unfortunately it doesn't work:

```python
_________________________________________________________________________ test_parsing_i_dont_control_code __________________________________________________________________________

option_parser = <flake8.options.manager.OptionManager object at 0x7f88c5880e50>

    def test_parsing_i_dont_control_code(option_parser):
        """Ensures that ``i_dont_control_code`` can be parsed."""
>       args, _ = option_parser.parse_args(['--i-dont-control-code'])
E       TypeError: cannot unpack non-iterable Namespace object

tests/test_options/test_option_values/test_i_control_code.py:21: TypeError
```

I couldn't get `Application` working either.

NB: see https://github.com/PyCQA/flake8/commit/f98d52a398cd2ff5cad270fdee9e37b62444550a for the relevant upstream commit.

Reply:
@sobolevn I found a solution. Please take a look :eyes: !